multi-node/aws/pkg/config/templates/stack-template.json

```diff
@@ -256,7 +256,11 @@
           {
             "DeviceName": "/dev/xvda",
             "Ebs": {
-              "VolumeSize": "{{.ControllerRootVolumeSize}}"
+              "VolumeSize": "{{.ControllerRootVolumeSize}}",
+              {{if gt .ControllerRootVolumeIOPS 0}}
```

Comment:
the integrity of this templating operation can be covered via unit tests in the `cluster` pkg.